docs/reference/elasticsearch/index.md

```diff
@@ -1,14 +1,55 @@
-# Elasticsearch and index management
+# Elasticsearch
 
-This section contains reference information for {{es}} and index management features.
+## Search and analyze your data
+----------------------------
 
-To learn more about {{es}} features and how to get started, refer to the [{{es}}](docs-content://solutions/search.md) documentation.
+Elasticsearch is the search and analytics engine that powers the Elastic Stack.
 
-For more details about query and scripting languages, check these sections:
-* [Query languages](../query-languages/index.md)
-* [Scripting languages](../scripting-languages/index.md)
- 
-{{es}} also provides the following REST APIs:
+[Get started](https://www.elastic.co/docs/get-started)
+
+[Introduction](https://www.elastic.co/docs/get-started) 
+
+[What's new](https://www.elastic.co/docs/release-notes/elasticsearch) 
+
+[Release notes](https://www.elastic.co/docs/release-notes/elasticsearch)
+
+![](https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/bltb8eb1c8cf2e7309e/636925fb7457f32a10457f6d/elasticsearch.png)
+
+## Get to know Elasticsearch
+
+_Demos:_ [Hands-on learning for Search](https://www.elastic.co/demo-gallery?solutions=search&features=null&type=hands-on-learning)
+
+_New webinar:_ [Architect search apps with Google Cloud](https://www.elastic.co/virtual-events/architecting-search-apps-on-google-cloud)
+
+[API quickstarts](docs-content://solutions/search/api-quickstarts.md)
+
+## Install, manage, and secure
```

Comment:
be very careful here. we shouldn't send the message that the only way to install elasticsearch is self-managed.